src/codec/builtin/FixSizedTypesCodec.ts

```diff
@@ -35,34 +40,35 @@ export class FixSizedTypesCodec {
         return buffer.readInt32LE(offset);
     }
 
-    static decodeLocalDate(buffer: Buffer, offset: number): string {
+    static decodeLocalDate(buffer: Buffer, offset: number): LocalDate {
         const year = FixSizedTypesCodec.decodeShort(buffer, offset);
```

Comment:
Shouldn't this be readInt?

Reply:
yeah somehow it is left like this, thanks for pointing out. Also to catch this, I will update DataTypeTest.js to use a year that is greater than the short range